docs/images/mocha-logo.svg

```diff
@@ -1,125 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Don't think you should have deleted the [XML declarations](https://stackoverflow.com/a/38172170).

Reply:
Of course they can be deleted.

Reply:
But see my original comment. The production SVGs are compressed with the svgo default atm.

And you will see this stuff is removed from there too.